Penteract/Assets/Scripts/RandomNumberGenerator.h

```diff
@@ -0,0 +1,23 @@
+#pragma once
+
+#include <random>
+
+class RandomNumberGenerator
+{
+public:
+	static int GenerateInteger(int min, int max) {
+		std::random_device rd;
```

Comment:
why create each time a random device and an engine? Since our seed is a non-deterministic number with hardware entropy source, we don't need to generate it each time. So I would recommend having the seed generator (the std::random_device) declared outside the methods. And the same goes for the Mersenne Twister pRNG.

Reply:
Since this is a static method, it would force me to declare the generator attribute as static, and Ihad no idea how that would behave, specially in a DLL, so I just went for a quick and safe method. If I have time, I might do some testing.

Reply:
Ok